app-backend/datamodel/src/main/scala/ColorCorrect.scala

```diff
@@ -310,4 +310,21 @@ object ColorCorrect extends LazyLogging {
   }
 
   private def lazyWrapper[T](f: => T): T = f
+
```

Comment:
What's this for?

Reply:
The `lazyWrapper` or the `paramsFromBandSpecOnly`?